be/src/olap/utils.h

```diff
@@ -281,19 +281,7 @@ struct RowLocation {
     uint32_t segment_id;
     uint32_t row_id;
 
-    bool operator==(const RowLocation& rhs) const {
-        return rowset_id == rhs.rowset_id && segment_id == rhs.segment_id && row_id == rhs.row_id;
```

Comment:
do not modify this, because when other people refactor  rowlocations, for example, change the order of rowset_id and segment_id, the result of operator < is changed.